app/extensions/brave/content/scripts/passwordManager.js

```diff
@@ -74,11 +74,19 @@ if (chrome.contentSettings.passwordManager == 'allow') {
             }, usernameElem.value || '')
         }
       })
+      usernameElem.addEventListener('blur', (e) => {
```

Comment:
Yeah, I cannot click on the pop-up context menu either.
But I got username/password filled when page loaded.